### PR TITLE
docs/rhcos: Update for new COSA feature

### PR DESCRIPTION
**Pull request by travier**
See: https://github.com/coreos/coreos-assembler/pull/3417

### CODE REVIEW

**Comment by travier** [comment]
Waiting on mentioned PR being merged.

**Review by jlebon — APPROVED** [review]
Thanks for updating these docs!

**Comment by travier** [comment]
@jlebon alright, I need a lgtm now :)

**Pull request merged**
